cargo-concordium/src/templates/default-lib.rs

```diff
@@ -0,0 +1,81 @@
+//! # A Concordium V1 smart contract
```

Comment:
Even if we don't add a submodule, these templates should be moved out of the `src` folder into, e.g., `data` or `resources` folder. `src` should be for sources of the project itself, i.e., the thing that gets compiled.